optaplanner-core/src/main/java/org/optaplanner/core/config/score/director/ScoreDirectorFactoryConfig.java

```diff
@@ -169,12 +171,22 @@ public Boolean getDroolsAlphaNetworkCompilationEnabled() {
         return droolsAlphaNetworkCompilationEnabled;
     }
 
+    @Deprecated
```

Comment:
```suggestion
        @Deprecated(forRemoval = true)
```

Also please add Javadoc along the lines of "@deprecated in favor of ...".